### PR TITLE
This is the customer model

### PR DESCRIPTION
**Pull request by AbelRodriguezGuillen**

### CODE REVIEW

File: NJInsurancePlatform/Models/CustomerModel.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace NJInsurancePlatform.Models
+{
+    public class CustomerModel
+    {
+        [Key]
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Models/CustomerModel.cs:8]: Add [ScaffoldColumn(false)] so we cannot edit primary key

Comment (Marcus-Johnson) [reply NJInsurancePlatform/Models/CustomerModel.cs:8]: You need to look at the current Policy Model https://github.com/Marcus-Johnson/NJInsurancePlatform/pull/4/files PolicyMUID is spelled differently and is a GUID

File: NJInsurancePlatform/Models/CustomerModel.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        public int CustomerMuid { get; set; }
+        [ForeignKey("PolicyModel")] // Change the argument here so that it matches the name of the policy model
+        [ScaffoldColumn(false)]
+        public int PolicyMuid { get; set; }
+        [Required(ErrorMessage = "Missing field: First Name")]
+        [StringLength(140)]
+        public string FirstName { get; set; }
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Models/CustomerModel.cs:15]: Add a DisplayName for each Data Type, or there will be no title in our view.

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]
Please make changes and review other Pull Requests.

File: NJInsurancePlatform/Models/CustomerModel.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace NJInsurancePlatform.Models
+{
+    public class CustomerModel
```
Comment (Marcus-Johnson) [inline NJInsurancePlatform/Models/CustomerModel.cs:6]: public partial Customer ?

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]

File: NJInsurancePlatform/Models/CustomerModel.cs (hunk 3 above)
Comment (AbelRodriguezGuillen) [reply NJInsurancePlatform/Models/CustomerModel.cs:6]: You want me to change it to a partial class?

File: NJInsurancePlatform/Models/CustomerModel.cs (hunk 2 above)
Comment (AbelRodriguezGuillen) [reply NJInsurancePlatform/Models/CustomerModel.cs:15]: Will do.

File: NJInsurancePlatform/Models/CustomerModel.cs (hunk 1 above)
Comment (AbelRodriguezGuillen) [reply NJInsurancePlatform/Models/CustomerModel.cs:8]: Okay, I will add that and make adjustments to the name. Thank you.

**Review by Marcus-Johnson — CHANGES_REQUESTED** [review]
public class Customer not customermodel because it effects the name of the generated table

**Pull request merged**